### PR TITLE
Replace Security Key with Passkey in WebAuthn UIs and their documents

### LINKED ISSUE #27147: Replace "Security Key" with "Passkey" in WebAuthn UIs and their documents
### Description

WebAuthn UIs (authentication, policy) and documentation about WebAuthn use the word "Security Key".
Recently, "Passkey" becomes used commonly instead of "Security Key".
Therefore, the issue is for replacing the word "Security Key" with "Passkey".

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by tnorimat**
closes #27147

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]
@tnorimat Thanks!

**Comment by mposolda** [comment]
@keycloak/ui-maintainers Anyone available to review UI changes in this PR please?

File: testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/WebAuthnRegisterAndLoginTest.java
Comment (jonkoops) [inline testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/WebAuthnRegisterAndLoginTest.java:265]:
```suggestion
            // Password + WebAuthn Passkey
```

File: testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/account/WebAuthnSigningInTest.java
Comment (jonkoops) [inline testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/account/WebAuthnSigningInTest.java:412]:
```suggestion
            expectedHelpText = "Use your Passkey for passwordless sign in.";
```

File: testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/account/WebAuthnSigningInTest.java
Comment (jonkoops) [inline testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/account/WebAuthnSigningInTest.java:416]:
```suggestion
            expectedHelpText = "Use your Passkey to sign in.";
```

File: themes/src/main/resources/theme/base/login/messages/messages_en.properties
Comment (jonkoops) [inline themes/src/main/resources/theme/base/login/messages/messages_en.properties:479]:
```suggestion
webauthn-passwordless-help-text=Use your Passkey for passwordless sign in.
```

File: themes/src/main/resources/theme/keycloak.v2/account/messages/messages_en.properties
Comment (jonkoops) [inline themes/src/main/resources/theme/keycloak.v2/account/messages/messages_en.properties:112]:
```suggestion
webauthn-passwordless-help-text=Use your Passkey for passwordless sign in.
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/apps/account-ui/maven-resources/theme/keycloak.v3/account/messages/messages_en.properties
Comment (tnorimat) [reply js/apps/account-ui/maven-resources/theme/keycloak.v3/account/messages/messages_en.properties:183]: I fixed it.

File: docs/documentation/server_admin/topics/account.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/account.adoc:72]: I fixed it.

Comment (tnorimat) [reply docs/documentation/server_admin/topics/account.adoc:72]: I fixed it.

File: docs/documentation/server_admin/topics/account.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/account.adoc:69]: I fixed it.

File: docs/documentation/server_admin/topics/account.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/account.adoc:74]: I fixed it.

File: docs/documentation/server_admin/topics/account.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/account.adoc:94]: I fixed it.

File: docs/documentation/server_admin/topics/account.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/account.adoc:97]: I fixed it. I will update the screenshot.

Comment (tnorimat) [reply docs/documentation/server_admin/topics/account.adoc:97]: I fixed it.

File: docs/documentation/server_admin/topics/account.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/account.adoc:99]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:165]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:194]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:204]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:228]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:239]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:254]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:257]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:258]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:262]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:265]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc:267]: I fixed it.

File: testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/WebAuthnIdlessTest.java
Comment (tnorimat) [reply testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/WebAuthnIdlessTest.java:268]: I fixe it.

File: testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/WebAuthnRegisterAndLoginTest.java
Comment (tnorimat) [reply testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/WebAuthnRegisterAndLoginTest.java:294]: I fixed it.

File: testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/WebAuthnRegisterAndLoginTest.java
Comment (tnorimat) [reply testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/WebAuthnRegisterAndLoginTest.java:265]: I fixed it.

File: testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/account/WebAuthnSigningInTest.java
Comment (tnorimat) [reply testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/account/WebAuthnSigningInTest.java:412]: I fixed it.

File: themes/src/main/resources/theme/base/login/messages/messages_en.properties
Comment (tnorimat) [reply themes/src/main/resources/theme/base/login/messages/messages_en.properties:479]: I fixed it.

File: themes/src/main/resources/theme/keycloak.v2/account/messages/messages_en.properties
Comment (tnorimat) [reply themes/src/main/resources/theme/keycloak.v2/account/messages/messages_en.properties:112]: I fixed it.

File: docs/documentation/server_admin/topics/authentication/webauthn.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/authentication/webauthn.adoc]: I fixed it.

File: docs/documentation/server_admin/topics/account.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/account.adoc]: I fixed it.

File: docs/documentation/server_admin/topics/account.adoc
Comment (tnorimat) [reply docs/documentation/server_admin/topics/account.adoc]: I updated the two screenshots.

**Comment by tnorimat** [comment]
@jonkoops Thank you for your review comments. I incorporated all of your review comments into the PR. Could you check them?

**Review by jonkoops — DISMISSED** [review]
LGTM!

**Review by mposolda — DISMISSED** [review]
@tnorimat @jonkoops Thanks for the updates and review! I hope to merge once tests are OK

**Comment by mposolda** [comment]
@tnorimat Could you please rebase? That should help with fixing the test failure in FIPS tests (it was fixed today in Keycloak main).

**Comment by tnorimat** [comment]
@mposolda I rebased the PR on the main branch.

**Review by ahus1 — APPROVED** [review]
re-approving after @mposolda's review and rebase

**Comment by ahus1** [comment]
@jonkoops - could you please re-approve after the rebase? Thanks!

**Review by mposolda — APPROVED** [review]
@tnorimat Thanks for the rebase!

**Pull request merged**
